### PR TITLE
add to node.js

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson2/node.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const f = require('fs');
+
+const read = (err,files)=>{
+  console.log(files);
```
Comment (songz) [inline lesson2/node.js]: good start! Now implement part 2:
Print out all the file names that are not you :)

File: lesson2/node.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//print out all the files in home - student names except for Kinue.
+
+const f = require('fs');
```
Comment (allopez7) [inline lesson2/node.js:3]: add a more descriptive name for f.

File: lesson2/node.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const read = (err,files)=>{
+  const notme=()=>{
+    let names = '';
+    files.forEach((e) => {
```
Comment (allopez7) [inline lesson2/node.js]: add a more descriptive name for e.

File: lesson2/node.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      if(e !== 'kinue'){
+        names = '\n' + names + '\n' + e;
```
Comment (allopez7) [inline lesson2/node.js]: get rid of the first '\n', no need for a new line initially.

File: lesson2/node.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      }
+    }); 
+    return names;
+  }
+  console.log(notme());
```
Comment (allopez7) [inline lesson2/node.js]: get rid of this console and notme() funtion call.

**Review by allopez7 — CHANGES_REQUESTED** [review]
I would also suggest adding more spacing to make it easier to read.

**Review by MadhuBeesetty — APPROVED** [review]
try giving a bit relative variable names so that you wont get confused while you code.

**Pull request merged**